models/comments.models.js

```diff
@@ -24,4 +24,17 @@ exports.insertComment = ({ body, username }, article_id) => {
     return db.query(queryStr, [body, username, article_id]).then(({ rows }) => {
         return rows[0]; 
     })
+}
+
+exports.removeCommentById = (comment_id) => {
+    return db.query(`
+        DELETE FROM comments
+        WHERE comment_id = $1
+        RETURNING*;`,
+        [comment_id])
+        .then(({ rows }) => {
+            if (rows.length === 0) {
+                return Promise.reject({ status: 404, message: 'Article Does Not Exist'})
+            }
+        })
```

Comment:
Looks good, nice use of $ syntax and a custom error for non-existent comment
Though I'd expect the error message to say "Comment does not exist" rather than article, no article_id has been provided